src/epgsql_wire.erl

```diff
@@ -209,6 +209,6 @@ encode_list(L) ->
     <<(byte_size(Bin)):?int32, Bin/binary>>.
 
 encode_standby_status_update(ReceivedLSN, FlushedLSN, AppliedLSN) ->
-    {MegaSecs, Secs, MicroSecs} = erlang:now(),
```

Comment:
monotonic versions are added on erlang 18, but epgsql supports versions from 15 to 19. So, this call is the only one supported on all VMs. We can try to use conditional compilation, but I don't think there is a reason for this extra complexity.